scripts/run-all-checks.sh

```
@@ -0,0 +1,380 @@
+# Default options
+PARALLEL=true
+RUN_CODE=false
+RUN_DOCS=false
+SCOPE_SPECIFIED=false
+
+# Get script directory and project root
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+# Virtualenv path: set VENV or VENV_PATH to override default project venv
+if [ -n "${VENV:-}" ]; then
+    VENV="${VENV}"
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)